webapp/lib/platform.dart

```diff
@@ -59,17 +59,23 @@ bool isUserSignedIn() {
 
 typedef CollectionListener(List<DocSnapshot> changes);
 
-StreamSubscription listenForMetrics(String collectionRoot, CollectionListener listener) {
+StreamSubscription listenForMetrics(String collectionRoot, DateTime periodFilterDatetime, String fieldPath, CollectionListener listener) {
```

Comment:
I think we should cancel the previous query on the same `collectionRoot` before we before we start a new one otherwise we'll receive the data multiple times, it's just we won't be doing anything with it.

Reply:
Already resolved.

Reply:
yes, thanks!